sentry-ruby/spec/sentry/cron/monitor_config_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'spec_helper'
+
+RSpec.describe Sentry::Cron::MonitorConfig do
+  describe '.from_crontab' do
+    it 'has correct attributes' do
+      subject = described_class.from_crontab(
+        '5 * * * *',
+        checkin_margin: 10,
+        max_runtime: 30,
+        timezone: 'Europe/Vienna'
+      )
+
+      expect(subject.schedule).to be_a(Sentry::Cron::MonitorSchedule::Crontab)
+      expect(subject.schedule.value).to eq('5 * * * *')
+      expect(subject.checkin_margin).to eq(10)
+      expect(subject.max_runtime).to eq(30)
+      expect(subject.timezone).to eq('Europe/Vienna')
+    end
+  end
+
+  describe '.from_interval' do
+    it 'returns nil without valid unit' do
```

Comment:
@sl0thentr0py would you consider having this raise instead? Maybe just in development/test?

I spent way too long trying to work out why this wasn't reporting properly:

![image](https://github.com/user-attachments/assets/c59553ba-16ec-42a1-983d-a8357f9d1f28)

Reply:
in general, as a principle, we don't raise in the Sentry SDK since we are an error monitoring service and we don't want to raise our own errors.

I could add debug logs if you wish.

Reply:
tracking https://github.com/getsentry/sentry-ruby/issues/2456